Makefile

```diff
@@ -169,19 +177,20 @@ seed:
 	python -m ledgerblue.hostOnboard --id $(NANOS_ID) --words $(WORDS) --pin $(PIN)
 
 
-# import generic rules from the sdk
-include $(BOLOS_SDK)/Makefile.rules
+##############
+#   Style    #
+##############
 
-#add dependency on custom makefile filename
-dep/%.d: %.c Makefile
+format:
+	astyle --options=.astylerc "src/*.h" "src/*.c" --exclude=src/glyphs.h --exclude=src/glyphs.c
 
-listvariants:
-	@echo VARIANTS COIN cardano_ada
 
 ##############
-#   Style    #
+#    Size    #
 ##############
 
-# better to run this manually to avoid irrelevant dependencies processing
-format:
-	astyle --options=.astylerc src/*.h src/*.c
+# prints app size, max is about 140K
+
+size: all
+	$(GCCPATH)arm-none-eabi-size --format=gnu bin/app.elf
```

Comment:
Interesting. With GNU size (GNU Tools for ARM Embedded Processors) 2.26.0.20160310, the --format does not accept gnu. --format={sysv|berkeley}

Reply:
I'm using GNU size (GNU Arm Embedded Toolchain 10-2020-q4-major) 2.35.1.20201028 and there it works. Perhaps we can use --format=berkeley, it's about the same as gnu, but I like "total" better than "dec" as column name.

Reply:
That's fine, I think we want to keep the up to date option.